src/components/select2/auiSelect2Mixin.js

```diff
@@ -73,8 +73,8 @@ export default {
     },
 
     mapToOriginalVal(stringValue) {
-      const original = this.options.find(option => `${option.value}` === stringValue);
-      return original && original.value
+      const original = this.options && this.options.find(option => `${option.value}` === stringValue);
```

Comment:
Can we ensure that the **options** field is never undefined so that no checks similar to this are required? Which would be putting an empty array in case default slot is empty in 68.